entity-framework/core/miscellaneous/connection-strings.md

```diff
@@ -93,3 +93,4 @@ public void ConfigureServices(IServiceCollection services)
         options.UseSqlServer(Configuration.GetConnectionString("BloggingDatabase")));
 }
 ````
+Note: the following using statement is required: using Microsoft.EntityFrameworkCore;
```

Comment:
@divega  

An easy way an unobtrusive way to address this (and one I'm using) is the add a comment
```
// Requires using Microsoft.EntityFrameworkCore;
options.UseSqlServer(Configuration.GetConnectionString("BloggingDatabase")));
  }
```

>Notes really should be quoted and start with a [!NOTE] header

We're trying to eliminate  [!NOTE]  and the other obtrusive dialogs. Compare [my rewrite](https://review.docs.microsoft.com/en-us/ef/core/get-started/netcore/new-db-sqlite?branch=Rick-Anderson-patch-1) to [the current version](https://docs.microsoft.com/en-us/ef/core/get-started/netcore/new-db-sqlite) of the New DB article.